RabbitMQ.Stream.Client/StreamSystem.cs

```diff
@@ -111,6 +111,18 @@ public async Task<bool> StreamExists(string stream)
             return response.StreamInfos is { Count: >= 1 } &&
                    response.StreamInfos[stream].ResponseCode == ResponseCode.Ok;
         }
+        private static void MaybeThrowQueryException(string reference, string stream)
+        {
+            if (string.IsNullOrEmpty(reference) || string.IsNullOrEmpty(stream))
+            {
+                throw new QueryException("Stream name can't be empty");
```

Comment:
I would change in:
```csharp
QueryException("Stream name reference  and can't be empty");
````

Reply:
Ok, 
I will set the message "Stream name and reference can't be empty or null".